chapters/14-team-package-dev.Rmd

```diff
@@ -1,19 +1,53 @@
 # R package development in a team-based environment
 
-- Making use of GitHub Organizations/Teams
-- Branch protection, member roles
-- Resolving conflict, personality differences
-- Running meetings, code of conduct
-- Decision making
-- Governance
-- Issue assignment
-- Relying on and using code of conduct to build culture and standards
-- Setting up SSH, two-factor authentication, PATs?
+-   Making use of GitHub Organizations/Teams
+-   Branch protection, member roles
+-   Resolving conflict, personality differences
+-   Running meetings, code of conduct
+-   Decision making
+-   Governance
+-   Issue assignment
+-   Relying on and using code of conduct to build culture and standards
+-   Setting up SSH, two-factor authentication, PATs?
 
+## Final exercises
 
+<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
-## Final exercise
+### For the first project assignment
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+These exercises are to be done on the *first project assignment* (the
+individual-based one). You won't be able to practice everything that you've
+learned during this chapter for this exercise.
+
+1. Set up branch protection for your main branch in your package repository on
+GitHub.
+2. Create a file called `GOVERNANCE.md` (FIXME: Should this be here?) and write
```

Comment:
Is "governance" the same as "team-working contract"? I think we could include more details about what goes into this, I don't know if this is the correct place for it, but here is something I wrote to my students that we can take whatever seems relevant from:


> A team work contract communicates specifically **how** the core group of people who are working together
and gives more detail about the logistics of working together and the expectations you have for each other.
This document will govern your working relationship and if done correctly,
should help you manage and resolve any issues that arise.
MIT's online software construction course has [a good description of what to put in a team work contract](http://web.mit.edu/6.005/www/fa15/projects/abcplayer/team-contract/).
Some key aspects of the team work contract could be:
>
> - How will work be distributed in a fair and equitable way?
>- What are the expected work hours for the project? 
>- How often will group meetings occur ([here is a nice article](http://third-bit.com/2018/05/11/meetings.html) on meetings)?
>- Will you have meeting agendas and minutes? If so, what will be the system for rotating through these responsibilities?
>- What will be the style of working?
>    - Will you start each day with [stand-ups](https://www.atlassian.com/agile/scrum/standups), or submit a summary of your contributions 4 hours before each meeting? or something else?
>- What is the quality of work each team member expects from themselves and each other?
>- When are team members not available (e.g., evenings and Sundays because of family obligations).
>- And any other similar things that govern your working relationships.
>
>Use this opportunity to use your prior knowledge/experience to improve your teamwork, communication, leadership, and organizational skills.
You will need all of these for your capstone projects (and beyond)!
```

Reply:
Yea, we'll include information on this within the content of the chapter. I'll take this text and wrap it in comments for now. Thanks!